app/queries/get_content_collection.rb

```diff
@@ -50,7 +50,7 @@ def editions
       scope = scope.where(document_type: document_types) if document_types.any?
       scope = scope.where(publishing_app:) if publishing_app
       scope = scope.with_document.where("documents.locale": locale) unless locale == "all"
-      scope = Link.filter_editions(scope, link_filters) if link_filters.present?
+      scope = Link.filter_editions(scope.with_document, link_filters) if link_filters.present?
```

Comment:
I think rather than having `scope.with_document` in a couple of places we should just update the initial scope to be `Edition.with_document` instead of `Edition.all` - as I can't imagine a scenario where we'd not query with a document anyway.

Reply:
Sure, I initially added another `join` as we don't technically need the `documents` join for certain queries but if you think it is unlikely then adding to the default scope makes the code more readable and less error prone 👍